### PR TITLE
First batch of longopt translation tests along with a few

### LINKED ISSUE #7743: Batch #1 of longoption tests
**Description of the desired feature**
First large batch of longoption translation tests including a few required changes to longopts .h files.

**Are you willing to help implement and maintain this feature?** Yes

### PR DESCRIPTION
**Pull request by rbdavis**
necessary changes to various longopts .h file.

**Description of proposed changes**
First large batch of longoptions translation tests, including a few necessary changes to longopts .h files.

Fixes #7743

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Review by remkos — APPROVED** [review]

File: src/longopt/movie_inc.h [hunk 1]
```diff
@@ -46,10 +49,10 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = { /* Local options for this m
 	          "f,g,p",                 "length,color,preserve" },
 	{ 0, 'L', "label",
 	          "e,s,f,p,c,t",           "time,string,frame,percent,col,textcol",
-	          "c,f,g,h,j,o,p,r,t",     "clearance,font,fill,shade,refpoint,offset,pen,rounded,format" },
+	          "s,c,f,g,h,j,o,p,r,t",   "scale,clearance,font,fill,shade,refpoint,offset,pen,rounded,format" },
 	{ 0, 'M', "master",
 	          "f,m,l",                 "first,middle,last",
-	          "r",                     "altdpu" },
+	          "r,v",                   "altdpu,view" },
```
Comment (PaulWessel) [inline src/longopt/movie_inc.h:55]: Better to just use dots_per_unit|dpu

**Comment by PaulWessel** [comment]
Let comment on this PR by Wednesday Oct 18. If you have no comments please at least just approve so I know if anyone is watching!

**Review by joa-quim — APPROVED** [review]

**Comment by joa-quim** [comment]
May I ask that next PR identify the longoptions proposals that differ from those of GMT.jl & PyGMT. As is, it's quite cumbersome to go check that manually.

**Comment by rbdavis** [comment]
> May I ask that next PR identify the longoptions proposals that differ from those of GMT.jl & PyGMT.

Right now I already have a few more modules in the pipeline which will get PRs as soon as the already-submitted PRs get cleared out. After those are submitted I will try to systematically comment future PRs as you have requested, Joaquim.

**Comment by joa-quim** [comment]
Hi Roger, that's fine, thanks.

**Comment by PaulWessel** [comment]
Last call, @GenericMappingTools/long-options-review. Tonight the hammer falls on this one.

File: src/longopt/movie_inc.h [hunk 2]
```diff
@@ -29,13 +29,16 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = { /* Local options for this m
...
 	          "",                      "",
 	          "n,p,s,w,W",             "nframes,tagwidth,first,wordsepall,wordseptab" },
+	{ 0, 'A', "audio",
+                  "",                      "",
+                  "e",                     "exact" },
 	{ 0, 'D', "displayrate",           "", "", "", "" },
 	{ 0, 'E', "title",
 	          "",                      "",
 	          "d,f,g",                 "duration,fadetime,fadecolor" },
 	{ 0, 'F', "videoformat",
```
Comment (PaulWessel) [inline src/longopt/movie_inc.h:39]: Just let it be format since not all outputs are video

Comment (PaulWessel) [reply src/longopt/movie_inc.h]: Done

**Comment by PaulWessel** [comment]
Accepted most changes.
I've made a few changes since **movie** options do not affect PyGMT or GMT.jl:

-C use --canvas instead of --canvassize
-F use _format_ rather than _videoformat_ since animated GIF is not really a movie.
-M use modifier|alias dotsperunit|dpu for clarity
-H use _subpixel_ instead of tempscale

I also updated movie-l2s.sh with these changes.  PR good to merge (I merged master into it to avoid any conflicts.

**Pull request merged**